enterprise/frontend/src/embedding-sdk-bundle/errors/error-docs-links.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Provide documentation links for each error codes.
+ * Custom fields get removed when serializing errors, so
+ * we need to store doc urls separately.
+ **/
+export const ERROR_DOC_LINKS: Record<string, string> = {
+  EXISTING_USER_SESSION_FAILED:
```

Comment:
I'd setup a proper key type to not shoot ourselves in the foot in the future. We don't know when we're going to change this error name or introduce a new one. This is a bit error prone.